docs/python/how-to-use-queries-in-python.md

```diff
@@ -0,0 +1,21 @@
+---
+id: how-to-use-queries-in-python
+title: How to use Queries in Python
+sidebar_label: Use Queries
+description: Use Queries
+tags:
+  - developer-guide
+  - sdk
+  - python
+---
+
+Use the `@workflow.query` decorator to define a method as a Query. Queries are just like Signals, but they should return a value.
```

Comment:
> Queries are just like Signals, but they should return a value.

I don't think we want to make this statement. We can say that "are defined just like signals", but otherwise they are not alike.